packages/signature-controller/src/SignatureController.ts

```diff
@@ -27,7 +27,26 @@ import {
 } from '@metamask/base-controller';
 import { Patch } from 'immer';
 import { AddApprovalRequest } from '@metamask/approval-controller';
-import { ApprovalType, ORIGIN_METAMASK } from '@metamask/controller-utils';
+
+const ORIGIN_METAMASK = 'metamask';
+enum ApprovalType {
+  AddEthereumChain = 'wallet_addEthereumChain',
+  EthDecrypt = 'eth_decrypt',
+  EthGetEncryptionPublicKey = 'eth_getEncryptionPublicKey',
+  EthSign = 'eth_sign',
+  EthSignTypedData = 'eth_signTypedData',
+  PersonalSign = 'personal_sign',
+  SwitchEthereumChain = 'wallet_switchEthereumChain',
+  Transaction = 'transaction',
+  WalletRequestPermissions = 'wallet_requestPermissions',
+  WatchAsset = 'wallet_watchAsset',
+  SnapDialogAlert = 'snap_dialog:alert',
+  SnapDialogConfirmation = 'snap_dialog:confirmation',
+  SnapDialogPrompt = 'snap_dialog:prompt',
+  Unlock = 'unlock',
+  ConnectAccounts = 'connect_accounts',
+  WalletConnect = 'wallet_connect',
+}
```

Comment:
These two are just here until we can safely update controller-utils in mobile right?

Reply:
Oh thanks, no I will remove them now, I was just doing it for my local patch for mobile :/